packages/stargate/src/signingstargateclient.ts

```diff
@@ -81,6 +91,18 @@ export interface SigningStargateClientOptions {
   readonly gasPrice?: GasPrice;
 }
 
+export function createDefaultTypes(prefix: string): AminoConverters {
```

Comment:
Do you need to export this? I guess it's only here for this one file. Other clients should create their own lists.